core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCatalog.java

```diff
@@ -69,10 +68,10 @@ public void testCreateTableBuilder() throws Exception {
             .withProperties(ImmutableMap.of("key2", "value2"))
             .create();
 
-    Assert.assertEquals(TABLE_SCHEMA.toString(), table.schema().toString());
-    Assert.assertEquals(1, table.spec().fields().size());
-    Assert.assertEquals("value1", table.properties().get("key1"));
-    Assert.assertEquals("value2", table.properties().get("key2"));
+    Assertions.assertThat(table.schema().toString()).isEqualTo(TABLE_SCHEMA.toString());
+    Assertions.assertThat(table.spec().fields()).hasSize(1);
+    Assertions.assertThat(table.properties().get("key1")).isEqualTo("value1");
```

Comment:
those should all be `.containsEntry` (please also double-check other places that are being modified as part of this PR so that we use the same mechanism everywhere)

Reply:
I double-checked and changed everything that could be changed to .containsEntry.